com.unity.render-pipelines.high-definition/Runtime/Debug/DebugDisplay.hlsl

```diff
@@ -17,6 +17,12 @@ static DirectionalShadowType g_DebugShadowAttenuation = 0;
 
 StructuredBuffer<int2>  _DebugDepthPyramidOffsets;
 
+// For quad overdraw, we need two halfscreen UAV (because a quad is 2x2 pixels). For vertex density, we need one fullscreen UAV.
+// These modes are exclusive so we make only one fullscreen allocation for both.
+// For vertex density, it stores the number of vertex projected in each pixel.
+// For quad overdraw, each 2x2 quad of the UAV contains the overdraw count in top-left pixel and the locked quad in the bottom-right pixel. The two other pixels of the quad are unused.
+RW_TEXTURE2D_X(uint, _DebugDisplayUAV) :register(u2);
```

Comment:
@sebastienlagarde  I don't remember the details of our platforms implementation, but for the explicit registers in the uav did it _need_ to be starting from after render target count or it is fine as soon as the register is _after_ the render target count?

Reply:
yes, I have think about that as well, guess the code may not work on PS4.
on PS4 we will need to bing on 2 + number of RT allocated. We will check in a second time (I tracked this).